### ISSUE TITLE
History.js is imported into the global namespace

### ISSUE THREAD

**Issue by nbrl** [digitalanimal/advocate-things-js-sdk#39]
It should be under `AT._utils.History`. Before fixing it, decide whether it would be worth rewriting the small subset we need due to the unusual behaviour in some circumstances.

**Comment by jabclab** [comment]
Seems to be [coupled to `window`](https://github.com/browserstate/history.js/blob/master/scripts/uncompressed/history.js#L2045). Also `History.js` appears to [include `json2.js` too](https://github.com/browserstate/history.js/blob/master/buildr.coffee#L53) so we're effectively including it twice at present.

**Comment by nbrl** [comment]
Yep, see issue #42. We might be best off rewriting the functionality we use. The alternative is to wrap History.js before we import it and either use the JSON bundled with it, or use the non-bundled History.js and ensure we pick up the other requirements it has.
